tests/Backend/CommonPart1/BucketsTest.php

```diff
@@ -47,8 +48,38 @@ public function testBucketsList()
     public function testBucketDetail()
     {
         $tokenData = $this->_client->verifyToken();
+        $displayName = "Romanov-Bucket";
         $bucket = $this->_client->getBucket($this->getTestBucketId());
         $this->assertEquals($tokenData['owner']['defaultBackend'], $bucket['backend']);
+
+        $bucketUpdateOptions = new BucketUpdateOptions($this->getTestBucketId(), $displayName);
+
+        $bucket = $this->_client->updateBucket($bucketUpdateOptions);
```

Comment:
přidat assertNotEquals ten změněný displayName

Reply:
Pridane

Reply:
Tohle jsem měl asi popsat líp - myslel jsem:

* assert že displayName není to na co ho měním (je něco jiného)
* změnit displayName (update)
* assert, že displayName je to co jsem updatnul a protože předtím byl něco jiného tak to funguje...

Reply:
Tak jak jsi to napsal ty je to jedno - buď spadne jeden assert nebo druhý. Pokud by to nefungovalo. Ale neřeší to situaci, kdy z nějakého důvodu ten bucket už to diplayName nastavené má na to co čekáš ještě než uděláš update...